include/libretro.h

```diff
@@ -1069,7 +1069,12 @@ enum retro_mod
                                            /* struct retro_midi_interface ** --
                                             * Returns a MIDI interface that can be used for raw data I/O.
                                             */
-#define RETRO_ENVIRONMENT_SET_ACTIVE_PORT_COUNT (49 | RETRO_ENVIRONMENT_EXPERIMENTAL)
```

Comment:
Just a comment on PR organization, but if you invert the order of these two commits you won't have a commit deleting an unrelated line.